src/app/space/create/deployments/deployments-donut/deployments-donut.component.ts

```diff
@@ -0,0 +1,84 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { debounce } from 'lodash';
+
+@Component({
+  selector: 'deployments-donut',
+  templateUrl: './deployments-donut.component.html',
+  styleUrls: ['./deployments-donut.component.less']
+})
+export class DeploymentsDonutComponent implements OnInit {
+
+  @Input() mini: boolean;
+  @Input() applicationId: string;
+
+  isIdled = false;
+  scalable = true;
+  pods: any;
+  desiredReplicas: number;
+
+  private scaleRequestPending = false;
+  private debounceScale: () => void;
+
+  constructor() { }
+
+  ngOnInit(): void {
+    this.debounceScale = debounce(this.scale, 650);
```

Comment:
Can't this be assigned when debounceScale is declared, and avoid setting that initial no-op lambda?